ci/ReleaseTool/Common.cs

```diff
@@ -1,14 +1,80 @@
-using NLog;
+using Newtonsoft.Json.Linq;
```

Comment:
I assume this library contains `NLog`'s functionality and some more?

Reply:
No this is the JSON library namespace, no longer using NLog namespace as it annoyed me. `NLogger.Logger Logger vs the to me confusing to read: Logger Logger`